functions/vercel/api/delete/movies.ts

```diff
@@ -0,0 +1,28 @@
+import 'isomorphic-fetch'
+import type { VercelRequest, VercelResponse } from '@vercel/node'
+import { withErrorHandler } from '../../_lib/helpers'
+// import { getXataClient } from '../../_lib/xata.codegen'
+
+async function handler(req: VercelRequest, res: VercelResponse) {
+  // const xata = getXataClient()
+  // const { id } = request.body
+
+  if (req.method !== 'DELETE') {
+    throw {
```

Comment:
Maybe not a big deal, but the problem of throwing something that is not an `Error` is that you don't get a stacktrace.